### PR TITLE
DOCS-3872 CI Results Explorer Documentation Edits

### PR DESCRIPTION
**Pull request by alai97**
### What does this PR do?

Adds two pages under the Explorer (Search and Search Syntax).

### Motivation

DOCS-3872

### Additional Notes

---

### Reviewer checklist
- [ ] Review the changed files.
- [ ] Review the URLs listed in the [Preview](#preview) section.
- [ ] Check images for PII
- [ ] Review any mentions of "Contact Datadog support" for internal support documentation.

### CODE REVIEW

File: content/en/synthetics/explorer/search_syntax.md [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+title: Search Syntax
+kind: documentation
+description: Learn how to create a search query in the CI Results Explorer.
+further_reading:
+- link: '/synthetics/explorer'
+  tag: 'Documentation'
+  text: 'Learn about the CI Results Explorer'
+---
+
+## Overview
```
Comment (peculiarism) [inline content/en/synthetics/explorer/search_syntax.md:11]: This is a really great suggestion to help customers with the query syntax!!

**Review by peculiarism — APPROVED** [review]

File: config/_default/menus/menus.en.yaml [hunk 2]
```diff
@@ -2680,10 +2680,20 @@ main:
     identifier: synthetics_testing_tunnel
     weight: 605
   - name: CI Results Explorer
-    url: /synthetics/ci_results_explorer
+    url: /synthetics/explorer
     parent: synthetics
-    identifier: synthetics_ci_results_explorer
+    identifier: synthetics_explorer
     weight: 7
+  - name: Search
```
Comment (kayayarai) [inline config/_default/menus/menus.en.yaml:2687]:
```suggestion
  - name: Search Test Batches
```

File: content/en/synthetics/explorer/search.md [hunk 3]
```diff
@@ -0,0 +1,76 @@
...
+| `Pipeline URL` | The pipeline URL associated with the batch. |
+| `Stage Name`   | The stage name associated with the batch.   |
+
+### Test result attributes
+
+**Test result** facets allow you to filter on your executed test results' attributes.
+
+| Facet            | Description                                                                                             |
+|------------------|---------------------------------------------------------------------------------------------------------|
+| `Execution Rule` | The execution rule associated with the test result of the batch: `Blocking`, `Non Blocking`, and `Skipped`. |
```
Comment (kayayarai) [inline content/en/synthetics/explorer/search.md:50]:
```suggestion
| <code>Execution&nbsp;Rule</code> | The execution rule associated with the test result of the batch: `Blocking`, `Non Blocking`, and `Skipped`. |
```
lol this drove me batty

File: content/en/synthetics/explorer/search_syntax.md [hunk 4]
```diff
@@ -0,0 +1,38 @@
...
+
+A query is composed of terms and operators. 
+
+There are two types of terms:
+
+- A **single term** is a single word such as `test` or `hello`.
+- A **sequence** is a group of wods surrounded by double quotes, such as `"hello dolly"`.
```
Comment (kayayarai) [inline content/en/synthetics/explorer/search_syntax.md:18]:
```suggestion
- A **sequence** is a group of words surrounded by double quotes, such as `"hello dolly"`.
```

File: content/en/synthetics/explorer/search_syntax.md [hunk 5]
```diff
@@ -0,0 +1,38 @@
...
+
+To combine multiple terms into a complex query, you can use any of the following Boolean operators:
+
+| Operator | Description                                                                                        |
+|--------------|------------------------------------------------------------------------------------------------------- |
+| `AND`        | **Intersection**: Both terms are in the selected views (if nothing is added, AND is taken by default). |
```
Comment (kayayarai) [inline content/en/synthetics/explorer/search_syntax.md:24]:
```suggestion
| `AND`        | **Intersection**: Both terms are in the selected views. If no operator is used, `AND` is used by default. |
```

**Review by kayayarai — CHANGES_REQUESTED** [review]
Just some little things :)

File: content/en/synthetics/explorer/search.md [hunk 6]
```diff
@@ -0,0 +1,76 @@
...
+| `Test ID`        | The test ID associated with the test result of the batch.                                               |
+| `Test Name`      | The test name associated with the test result of the batch.                                             |
+
+### Git attributes
+
+**Git** facets allow you to filter on your batches' Git-related attributes.
+
+| Facet       | Description                               |
+|-------------|-------------------------------------------|
+| `Branch`    | The branch associated with the batch.     |
```
Comment (peculiarism) [inline content/en/synthetics/explorer/search.md:62]: @alai97 Last minute change, could we also add the `Author Email` facet? We added it quite recently, and it corresponds to the commit author

**Review by kayayarai — APPROVED** [review]

**Pull request merged**